### PR TITLE
Updates the SCIM Reference documents to clarify behaviour with OIN vs AIW app integrations

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?**
Updated the reference files for V1.1 and V2.0 for the PUT vs PATCH method requests when updating User and Group objects.

Also unified the language for "method requests" as that is the w3 standard for referring to the operations carried out over HTTP.

- **Is this PR related to a Monolith release?**
No

### Resolves:

* [OKTA-317115](https://oktainc.atlassian.net/browse/OKTA-317115)
* [OKTA-311445](https://oktainc.atlassian.net/browse/OKTA-311445)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 1]
```diff
@@ -330,7 +330,12 @@ Content-Type: text/json;charset=UTF-8
 
 **PUT** /Users/*$userID*
 
-After the user's profile is retrieved from the SCIM server, Okta modifies the attributes that were changed and runs a PUT request with the new body to the `/Users/${userID}` endpoint:
+>**Note:**
+>
+> * For any new OIN app integrations, all updates to a User profile are handled using a PUT method request, except as noted [below](#update-a-specific-user-profile-patch).
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:335]: We should avoid describing text locations like "below".

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 2]
```diff
@@ -396,21 +401,23 @@ Content-Type: text/json;charset=UTF-8
...
-### Update a specific User (PATCH)
+### Update a specific User profile (PATCH)
 
-![Flowchart - update User (PATCH)](/img/oin/scim_flow-user-update-patch.png "Simple flow diagram for updating a User with a PATCH operation")
+![Flowchart - update User (PATCH)](/img/oin/scim_flow-user-update-patch.png "Simple flow diagram for updating a User with a PATCH method request")
 
 **PATCH** /Users/*$userID*
 
-A User object can be updated through a PATCH operation for the following actions:
+For new OIN app integrations, the following operations update a User object through a PATCH method request:
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:410]: "User object" or "User profile object"?

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 3]
```diff
@@ -396,21 +401,23 @@ Content-Type: text/json;charset=UTF-8
...
 
 * Activating a User
 * Deactivating a User
 * Syncing the User password
 
-The `active` attribute in a user profile represents the user's current status.
+All other updates to User objects should be handled through a PUT method request.
+
+For any custom app integrations created using the AIW, all SCIM operations that update a User object, including these operations, are always sent through a PUT method request.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:418]: Same question - the User, or the User profile?

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md [hunk 4]
```diff
@@ -237,13 +237,13 @@ Content-Type: text/json;charset=UTF-8
 }
 ```
 
-### Update a specific User (PUT)
+### Update a specific User profile (PUT)
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:240]: Same confusion here - User object, or User profile object?

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md (hunk 2 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:410]: Added text to clarify "user application profile" (Okta side) vs "User object" (SCIM server side) throughout.

**Pull request merged**